PowerEditor/src/localization.cpp

```diff
@@ -477,52 +467,50 @@ void NativeLangSpeaker::changeMenuLang(HMENU menuHandle)
 			pos = z;
 		}
 
-		const wchar_t *nameW = wmc.char2wchar(name, _nativeLangEncoding);
+		const wchar_t* nameW = wmc.char2wchar(name, _nativeLangEncoding);
 		::ModifyMenu(hMenu, pos, MF_BYPOSITION, 0, nameW);
 	}
 }
 
 
-static const int tabCmSubMenuEntryPos[] =
-{
+static constexpr int tabCmSubMenuEntryPos[]{
 //   +-------------- The submenu entry item position on the top level of tab context menu
 //   |
 //   |       +------- Index order (CMDID: Context Menu submenu entry ID): in <TabBar> of english.xml - the number and the order of this array should be synchronized with <TabBar>
 //   |       |
 //   |       |
 //   |       |
-     1,   // 0  Close Multiple Tabs
-     5,   // 1  Open into
-    14,   // 2  Copy to Clipboard
-    15,   // 3  Move Document
-    16,   // 4  Apply Color to Tab
+	 1,   // 0  Close Multiple Tabs
+	 5,   // 1  Open into
+	14,   // 2  Copy to Clipboard
+	15,   // 3  Move Document
+	16,   // 4  Apply Color to Tab

```

Comment:
Due to the length of TAB symbol could be variable, please restore the white space of the above line.